modules/crawler.py

```diff
@@ -155,15 +177,16 @@ def crawl(self):
                     sys.stdout.write("-- Results: " + str(len(ord_lst)) + "\r")
                     sys.stdout.flush()
 
-                # Pause time.
-                if float(self.c_pause) > 0:
```

Comment:
Keep this condition as it is foolproof.

Reply:
c_pause has a default value of 1, hence the if statement felt unnecessary.